src/main/java/kg/jarkyn/cobspecserver/responders/RedirectResponder.java

```diff
@@ -0,0 +1,36 @@
+package kg.jarkyn.cobspecserver.responders;
+
+import kg.jarkyn.cobspecserver.data.Request;
+import kg.jarkyn.cobspecserver.data.Response;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static kg.jarkyn.cobspecserver.utils.Status.REDIRECT;
+
+public class RedirectResponder extends Responder {
+    private static final String DOMAIN_URL = "http://localhost:5000";
```

Comment:
I'm wondering if there's a way to remove reliance on domain + port in this responder. You have defined the port explicitly in your runner. If this were to change, it can break this responder.